Feature/SamvritSrinath/Landing-Page

SamvritSrinath opened the pull request:
### Tracking Info

Resolves #<issue number>

Make sure your branch name conforms to: `<feature|staging|bugfix|...>/<username>/<3-4 word description separated by dashes>`. Otherwise, please rename your branch and create a new PR.

### Changes

We created the first landing static page without the NavBar and have a separate page associated with it that is rendered in App.tsx. The images present are static and don't link to anywhere as of yet: and are updated with the images presented in Figma MVP. 

### Testing

Compared the first site to the figma, and compared it to scale to ensure that users would be able to see all the details present within the site.

### Confirmation of Change 

<img width="1426" alt="Screenshot 2023-01-28 at 2 42 04 PM" src="https://user-images.githubusercontent.com/114899512/215294411-a7055c13-7a56-44ce-af06-f1490cbd675b.png">
<img width="1426" alt="Screenshot 2023-01-28 at 2 42 12 PM" src="https://user-images.githubusercontent.com/114899512/215294417-c6599bf4-7c75-4178-9bcb-2751b5d6b6a1.png">

SamvritSrinath committed:
Basic React for Landing Page

Used JSX(will return to EM for feedback)

SamvritSrinath committed:
modified:   ../.DS_Store
	modified:   package-lock.json
	modified:   package.json
	modified:   src/App.jsx
	modified:   src/components/Body.jsx
	modified:   src/components/NavBar.jsx
	src/Images/

SamvritSrinath committed:
new file:   SDCTA/src/Images/Graph.jpg
	new file:   SDCTA/src/Images/SDTEF Logo Transparent Background 1.jpg
	new file:   SDCTA/src/Images/unsplash_dsV4yvL5UJk.jpg

meganleongg committed:
changed jsx to tsx

meganleongg committed:
added separate css file for body component

SamvritSrinath committed:
modified:   src/App.css
	modified:   src/App.tsx
	new file:   src/components/Footer.css
	modified:   src/components/Footer.tsx
	deleted:    src/components/NavBar.jsx

SamvritSrinath committed:
Merge branch 'Landing-Page' of https://github.com/TritonSE/SDCTA-Data-Visualization-Website into Landing-Page

SamvritSrinath committed:
Images in Body

SamvritSrinath committed:
Added Education Images
	modified:   src/components/Body.tsx

meganleongg committed:
changed heading

meganleongg committed:
Merge branch 'Landing-Page' of https://github.com/TritonSE/SDCTA-Data-Visualization-Website into Landing-Page

SamvritSrinath committed:
Updating Footer Style
	modified:   src/components/Footer.css

SamvritSrinath committed:
Updating Footer Style again
	modified:   src/components/Footer.css

SamvritSrinath committed:
Fixing white space
	modified:   src/components/Footer.css
	modified:   src/components/body.css
	new file:   src/pages/LandingPage.tsx

meganleongg committed:
put in new images & added image styling

meganleongg committed:
pull
Merge branch 'Landing-Page' of https://github.com/TritonSE/SDCTA-Data-Visualization-Website into Landing-Page

SamvritSrinath committed:
pulling from Megan
Merge branch 'Landing-Page' of https://github.com/TritonSE/SDCTA-Data-Visualization-Website into Landing-Page

SamvritSrinath committed:
Formatting into a page
	modified:   src/App.tsx
	modified:   src/pages/LandingPage.tsx

NirmalAgnihotri commented on SDCTA/src/components/Body.tsx:
```diff
@@ -0,0 +1,56 @@
+				SDTEF Data Museum
+			</div>
+			<div className="description" >
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
```
Minor Point. Not sure how this alignment is showing up in vs code but you can use something like the Prettier [extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) . You can then enable Vscode to format with Prettier on [save](https://www.alphr.com/auto-format-vs-code/#:~:text=Click%20the%20%E2%80%9CSettings%E2%80%9D%20gear%20icon,%3A%20Format%20On%20Save%E2%80%9D%20option.)

NirmalAgnihotri commented on SDCTA/src/components/Body.tsx:
```diff
@@ -0,0 +1,56 @@
+
+export const Body = () => {
+	return (
+		<><div>
```
Similar to comment below, we probably don't want the fragment to have the same alignment as the div because then its harder to tell that the fragment is actually the root, i.e. the top level tag.

NirmalAgnihotri commented on SDCTA/src/components/Footer.css:
```diff
@@ -0,0 +1,67 @@
+/* Footer Styling */
```
We probably want body.tsx, footer.tsx, body.css, etc. to be in one folder called LandingPage .

Anthonyp0329 replied:
agreed, with the exception of footer.tsx and footer.css as these are uniform, should probably just go in their own folder called "Footer"

NirmalAgnihotri reviewed (approved):
Great work! Code looks good, just left a few minor comments about formatting and structure.

Anthonyp0329 commented on SDCTA/src/components/Body.tsx:
```diff
@@ -0,0 +1,56 @@
+			</div>
+			<img src={landscape} alt="sd landscape" />
+		</div>
+			<div className="box" >
```
Only issue I see is maybe more descriptive classnames than "box" or "one", something more along the lines of "preview_box" and "first_visual"

Anthonyp0329 commented on SDCTA/src/components/Body.tsx:
```diff
@@ -0,0 +1,56 @@
+import landscape from '../Images/unsplash_dsV4yvL5UJk.jpg';
```
May be good to rename this file and its corresponding CSS file to something like LandingBody for less confusion (even if its put in its own separate folder)

Anthonyp0329 reviewed (approved):
A few more minor changes here, but looking great! Check comments, mostly to do with file naming/organization.

SamvritSrinath committed:
Image Refactor
	modified:   src/App.css
	deleted:    src/Images/unsplash_dsV4yvL5UJk.jpg
	deleted:    src/components/Body.tsx
	deleted:    src/components/body.css
	modified:   src/pages/LandingPage.tsx

SamvritSrinath committed:
Landing Page_Body Refactor

SamvritSrinath committed:
Images
	renamed:    src/components/Landing Page/Body.tsx -> src/components/Landing_Page/Body.tsx
	renamed:    src/components/Landing Page/body.css -> src/components/Landing_Page/body.css
	modified:   src/pages/LandingPage.tsx

SamvritSrinath committed:
Final Images
	renamed:    src/components/Footer.css -> src/components/Footer/Footer.css
	renamed:    src/components/Footer.tsx -> src/components/Footer/Footer.tsx
	modified:   src/components/Landing_Page/Body.tsx
	modified:   src/pages/LandingPage.tsx

SamvritSrinath commented:
Created Basic Landing Page for Website.
